features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -106,6 +112,11 @@ void AT_CellularContext::enable_hup(bool enable)
     }
 }
 
+void AT_CellularContext::do_connect_with_retry()
+{
+    CellularContext::do_connect_with_retry();
```

Comment:
Why do you override if you just directly call the base?

Reply:
It's because of EventQueue. In non blocking mode we use EventQueue to call do_connect_with_retry. We can't call CellularContext::do_connect_with_retry() as it's protected so to avoid that problem we had to introduce this function.